ReadMe.md

```diff
@@ -64,17 +89,22 @@ setInterval(function (msg) {
 }, 5000, 'Hello world');
 ```
 
-**timer.clear() / interval.clear()**
+**Interval.clear() // clearInterval(Interval interval)**
```

Comment:
I'd do a lowercase "i" on the `interval.clear()`.  The uppercase "I" would suggest that it is a static method on the `Interval` class.

Also, just one "/" between the two, please.